modules/govuk/manifests/apps/frontend.pp

```diff
@@ -33,14 +38,15 @@
   govuk::app { 'frontend':
     app_type               => 'rack',
     port                   => $port,
-    vhost_aliases          => ['private-frontend'],
+    vhost_aliases          => ['private-frontend', 'draft-frontend'],
```

Comment:
We thought that it was required for routing of requests for draft-frontend? If that's not the case we can remove it.